src/components/ble/NotificationManager.h

```diff
@@ -30,13 +32,16 @@ namespace Pinetime {
         Id id = 0;
         bool valid = false;
         uint8_t size;
+        std::time_t timeArrived;
         std::array<char, MessageSize + 1> message;
         Categories category = Categories::Unknown;
 
         const char* Message() const;
         const char* Title() const;
       };
 
+      NotificationManager(Controllers::DateTime& dateTimeController) : dateTimeController {dateTimeController} {
```

Comment:
we can even store a `const` reference, to make it explicit, that a notification can't change the date-time, just get the current date and time
```suggestion
      NotificationManager(const Controllers::DateTime& dateTimeController) : dateTimeController {dateTimeController} {
```

Reply:
At present it is not possible since DateTime controller updates it member variables during time retrieving.
https://github.com/InfiniTimeOrg/InfiniTime/blob/d69cfcfb136e45486437ad773d362da7870042e3/src/components/datetime/DateTimeController.cpp#L72-L102

It can be overcomed with `mutable` but I am not sure whether this is a good idea.